src/instructlab/model/chat.py

```diff
@@ -585,6 +587,32 @@ def _handle_list_contexts(self, _):
         self._sys_print(Markdown(f"**Available contexts:**\n\n{context_list}"))
         raise KeyboardInterrupt
 
+    def _handle_set_custom_context(self, content):
+        parts = content.split(" ", 2)
```

Comment:
please `rstrip` the input first, otherwise you allow to pass `/sc test_context ` (note the trailing space).